pulumi/grapl/__main__.py

```diff
@@ -135,6 +135,12 @@ def main() -> None:
 
         kafka = Kafka("kafka")
 
+        # These are created in `grapl-local-infra.nomad` and not applicable to prod.
+        # Nomad will replace the LOCAL_GRAPL_REPLACE_IP sentinel value with the correct IP.
+        aws_endpoint = "http://LOCAL_GRAPL_REPLACE_IP:4566"
+        kafka_endpoint = "LOCAL_GRAPL_REPLACE_IP:19092"  # intentionally not 29092
```

Comment:
I _think_ that these may need to move to Pulumi.local-grapl.yaml in the future since presumably we'll be passing in kafka and redis endpoint info to live services

Reply:
I wasn't sure, since Redis is actually provisioned BY pulumi